args.go

```diff
@@ -322,7 +359,12 @@ func copyArgs(dst, src []argsKV) []argsKV {
 		dstKV := &dst[i]
 		srcKV := &src[i]
 		dstKV.key = append(dstKV.key[:0], srcKV.key...)
-		dstKV.value = append(dstKV.value[:0], srcKV.value...)
+		if ArgsNoValue == srcKV.noValue {
```

Comment:
I would prefer `srcKV.noValue == ArgsNoValue` which reads easier and is more in line with the rest of the codebase.